### DISCUSSION TITLE
Scanning SWATH Q1 calibration

### DISCUSSION THREAD

**Discussion by DavidMSM** [vdemichev/DiaNN#169] [General]
Hi,

I really enjoyed your Nature Biotech Paper. I wondered whether the calibration of the masses using residual precursors in MS2 scans is a function of DIA-NN or whether this is part of the Sciex Scanning SWATH algorithm? I'm working with Waters SONAR data and as far as I understand I could use these in DIA-NN as generic DIA data. However, to fully exploit the Q1 dimension of scanning quadrupol DIA, I'd need to encode the center mass of each individual Q1 bin in the data. Is that right?

best wishes,
David

**Comment by vdemichev** [comment]
Hi David,

Sorry for the late reply, I somehow miss posts sometimes.
DIA-NN can perform a basic Q1 calibration, but this is not really necessary and is currently implemented in the Sciex acquisition software.
To enable Q1 calibration in DIA-NN use --q1-cal, also need to force DIA-NN to consider the run as Scanning SWATH using the --scanning-swath command, if analysing non-Sciex data. But I've never tested it on Waters SONAR, so cannot say anything about whether it will work or not.

Best,
Vadim
